Mathmagician/Integer.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Mathmagician
+{
+    public class Integer
+    {
+        public int newInteger = -1;
+        public List<int> integers = new List<int>();
+        public void GenerateIntegers(int numOfIntegers)
```

Comment:
If you are unit testing this method, you should unit test that the list is being added to as expected. Test all the functionality this outputs